composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": ">=2.0,<3.0",
+        "symfony/framework-bundle": ">=2.0,>=3.0",
```

Comment:
Please change to ">=2.0,<=4.0" to all compatibility with Symfony 2. Like this it is allowed to be used only with Symfony 3.